plugin/minimal/README.md

```diff
@@ -0,0 +1,31 @@
+# minimal-responses
```

Comment:
Just name it `minimal` and drop the `-responses` part, this can be done throughout

Reply:
done